boskos/OWNERS

```diff
@@ -1,3 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
 approvers:
 - krzyzacy
 - sebastienvas
+labels:
+- area/boskos
```

Comment:
IMO the OWNERS changes should be in their own PR. Right now the label sync utility only runs once a day.

Reply:
these labels all exist already, so it's not like the utility is adding them, just their descriptions, otherwise yeah I would break this up

and in this case I'd like to run the manual job to confirm this takes before falling back to leaning on the cronjob